stack/core/src/test/java/org/apache/usergrid/persistence/CollectionDeleteTest.java

```diff
@@ -42,6 +43,7 @@
 
 
 @NotThreadSafe
+@Ignore("fix later")
```

Comment:
Can we get a JIRA ticket opened with detail only this and link the ignore comment to the JIRA ticket.

Reply:
Created JIRA ticket https://issues.apache.org/jira/browse/USERGRID-1348
Fixed the issue as well.